### PR TITLE
Variadic argument lists

### PR DESCRIPTION
**Pull request by LPeter1997**

### CODE REVIEW

File: src/Draco.Compiler/Internal/Binding/BinderFacts.cs [hunk 1]
```diff
@@ -1,7 +1,9 @@
 using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
 using Draco.Compiler.Api.Syntax;
 using Draco.Compiler.Internal.Symbols;
 using Draco.Compiler.Internal.Symbols.Source;
+using Draco.Compiler.Internal.Symbols.Synthetized;
```
Comment (Binto86) [inline src/Draco.Compiler/Internal/Binding/BinderFacts.cs:6]: Are all those usings needed?

File: src/Draco.Compiler/Internal/Lowering/LocalRewriter.cs [hunk 2]
```diff
@@ -1,8 +1,11 @@
...
 using System.Collections.Generic;
 using System.Collections.Immutable;
 using System.Linq;
+using System.Reflection;
 using System.Text;
 using Draco.Compiler.Api;
+using Draco.Compiler.Internal.Binding;
 using Draco.Compiler.Internal.BoundTree;
 using Draco.Compiler.Internal.Symbols;
 using Draco.Compiler.Internal.Symbols.Synthetized;
```
Comment (Binto86) [inline src/Draco.Compiler/Internal/Lowering/LocalRewriter.cs:11]: Again, do we need all the usings?

File: src/Draco.Compiler/Internal/Binding/BinderFacts.cs (hunk 1 above)
Comment (LPeter1997) [reply src/Draco.Compiler/Internal/Binding/BinderFacts.cs:6]: Yes

File: src/Draco.Compiler/Internal/Lowering/LocalRewriter.cs (hunk 2 above)
Comment (LPeter1997) [reply src/Draco.Compiler/Internal/Lowering/LocalRewriter.cs:11]: Almost, reflection was redundant

**Pull request merged**